### DISCUSSION TITLE
keep-alive

### DISCUSSION THREAD

**Discussion by LForLuketastic** [NginxProxyManager/nginx-proxy-manager#1623] [Q&A]
Hello together

Is it possible to set keep-alive in NPM? I have to create a stream on an incoming port to an IP address with a further forwarded port. When accessing through this port with the suffix /keep-alive a blank page should be displayed, but then it says "Cannot GET /keep-alive". I also have a proxy host with the FQDN on the host but this does not work either. What am I doing wrong?

**Comment by MiranoVerhoef** [comment]
I would like to add to this,

i really want to improve my speed on certain applications and it seems like KeepAlive would contribute to that

com.github.nginx-proxy.nginx-proxy.keepalive

Is there a docker variable to enable this?
